ducktape/tests/reporter.py

```diff
@@ -18,45 +18,109 @@
 import pkg_resources
 
 
-class TestReporter(object):
+SEPARATOR_LENGTH = 100
```

Comment:
You might just want something like [this](https://gist.github.com/jtriley/1108174) instead of using a fixed width. Although maybe that only makes sense for the Stdout versions.